llvm/lib/CodeGen/TargetLoweringObjectFileImpl.cpp

```diff
@@ -212,13 +212,8 @@ void TargetLoweringObjectFileELF::Initialize(MCContext &Ctx,
     //        identify N64 from just a triple.
     TTypeEncoding = dwarf::DW_EH_PE_indirect | dwarf::DW_EH_PE_pcrel |
                     dwarf::DW_EH_PE_sdata4;
-    // We don't support PC-relative LSDA references in GAS so we use the default
-    // DW_EH_PE_absptr for those.
 
-    // FreeBSD must be explicit about the data size and using pcrel since it's
-    // assembler/linker won't do the automatic conversion that the Linux tools
-    // do.
```

Comment:
What's changed to make neither of these comments correct?

Reply:
You are right. We should be careful. Let's keep the OSFreebsd here.